browser/mapping.go

```diff
@@ -689,8 +704,22 @@ func mapPage(vu moduleVU, p *common.Page) mapping {
 			return rt.ToValue(r).ToObject(rt)
 		},
 		"route": p.Route,
-		"screenshot": func(opts goja.Value) goja.ArrayBuffer {
-			return p.Screenshot(opts, vu.LocalFilePersister)
+		"screenshot": func(opts goja.Value) (*goja.ArrayBuffer, error) {
+			ctx := vu.Context()
+
+			popts := common.NewPageScreenshotOptions()
+			if err := popts.Parse(ctx, opts); err != nil {
+				return &goja.ArrayBuffer{}, fmt.Errorf("parsing page screenshot options: %w", err)
```

Comment:
Also, here: `nil`?

Reply:
Resolved in 11219b4572a8bcfd5eecd4acedfa130291de2979